live-examples/html-examples/text-content/css/main.css

```diff
@@ -1,5 +1,5 @@
 header {
   font:
-    bold calc(0.025 * (100vw)) Arial,
+    bold calc(0.05 * (100vw)) Arial,
```

Comment:
Not sure why this can't be:

```suggestion
    bold 5vw Arial,
```

...isn't it the same?

Reply:
nice👌, it is the same. Thanks for this suggestion, I was like a mechanic just starting an engine in the middle of the desert. 😂

Also, I made changes in the size 7vw of the font, because 5vw is too small on small screens.
 
![main](https://github.com/mdn/interactive-examples/assets/91728992/3b13c997-f0b9-4e9d-ac8f-14d6223af3d7)

Reply:
Thank you @ShubhamOulkar :) I agree 7vw is better.